.github/workflows/ci.yml

```diff
@@ -68,13 +68,3 @@ jobs:
       test_path: test_all_seq
       env: acl
       use_akv: ${{ matrix.use_akv }}
-
-
-  standalone-image:
```

Comment:
Do we still have a CI that tests against local KMS after this PR?

Reply:
Yes if you look at all the testing on this PR, all but 2 of the tests are with a local KMS, the removed test here was checking that the special image used by privacy-sandbox-dev (which runs CCF and sets up the KMS in one image instead of running a CCF sandbox container then configuring it later) builds

Reply:
We remove that special image now because privacy-sandbox-dev now does what the rest of the KMS tests do, run CCF (either with a local container calling sandbox.sh, or ACL) and then sets up the KMS against it